pyproject.toml

```diff
@@ -163,6 +163,7 @@ ignore = [
 # Ignore `E402` and `F401` (unusued imports) in all `__init__.py` files
 "__init__.py" = ["E402", "F401"]
 "tests/*" = ["S101"] # Ignore assert statements in tests
+"pycytominer/cyto_utils/*" = ["S101", "S608"] # Ignore assert statements in cyto_utils
```

Comment:
Double checking: with the `noqa` comment additions do you still need this line? I suggest keeping one or the other, but not both if we can help it.

Reply:
The `noqa` comments were covering things outside of `cyto_utils`. I got rid of the path-based ignore for `S101` though and added additional noqa statements to all those lines. Those asserts are relatively easy to tackle one at a time and we want to make sure we don't add more. 

I left the `S608` as a path-based ignore because it may require a larger effort to change the pattern that we use to construct SQL queries.